platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheFileSystemDefaultExcludesTest.groovy

```diff
@@ -68,11 +69,7 @@ class ConfigurationCacheFileSystemDefaultExcludesTest extends AbstractConfigurat
         def excludedByBuildScriptFileCopy = file("build/output/${excludedByBuildScriptFileName}")
         excludedByBuildScriptFile.text = "input"
 
-        if (scriptLanguage == DefaultExcludesFixture.ScriptLanguage.KOTLIN) {
-            buildKotlinFile << DefaultExcludesFixture.DefaultExcludesLocation.addDefaultExclude(excludedByBuildScriptFileName)
-        } else {
-            buildFile << DefaultExcludesFixture.DefaultExcludesLocation.addDefaultExclude(excludedByBuildScriptFileName)
-        }
+        getBuildFile(dsl) << DefaultExcludesFixture.DefaultExcludesLocation.addDefaultExclude(excludedByBuildScriptFileName)
```

Comment:
I am quite surprised to see we support default excludes in DCL. 

Do you know the reasoning behind this?

Reply:
No, sorry, the PR also contains some "clean-up for loosely related code", this being one of those cases. This test doesn't run/work with DCL files.

Reply:
Ah didn't realize this didn't run in DCL